libs/design/src/molecules/backdrop/backdrop/backdrop.component.scss

```diff
@@ -3,7 +3,6 @@
 :host {
 	@include clickable();
 	display: block;
-	-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```

Comment:
Let's prefer `rgba(0, 0, 0, 0)` over `transparent`

Reply:
Okay this should be resolved in the new commit.